doc/getting-started/installation.md

```diff
@@ -56,6 +56,10 @@ curl "https://github.com/jolicode/castor/releases/latest/download/castor.darwin-
 curl.exe "https://github.com/jolicode/castor/releases/latest/download/castor.windows-amd64.phar" -Lso C:\<a directory in your PATH>\castor
 ```
 
+> [!NOTE]
+> When using Windows Subsystem for Linux (WSL), you should still use the
+> Windows PHAR instead of the Linux PHAR.
+
```

Comment:
I would move this in the linux section, and use CAUTION instead of NOTE

Reply:
Done